05/DB.test.js

```diff
@@ -1 +1,140 @@
-import DB from './DB';
\ No newline at end of file
+import DB from './DB';
+
+describe('insert', () => {
+    // scenario
+    it('return data when ID is a number', () => {
+        // given
```

Comment:
Tutaj powinieneś dodać: `expect.assertions(1);`.
Inaczej w przypadku zwrócenia błędu w obietnicy nie zostanie wykonany test. To nie wywoła błędu.
Natomiast jak zadeklarujesz, że spodziewasz się przynajmniej jednego porównania to będzie błąd jeśli żadne nie zostanie wykonane.